src/main/java/domain/Car.java

```diff
@@ -0,0 +1,33 @@
+package domain;
+
+public class Car {
+    private String name;
+    private int position;
+
+    //실제 사용되는 생성자
+    public Car(String name) {
+        this.name = name;
+        position = 0;
+    }
```

Comment:
```suggestion
    //실제 사용되는 생성자
    public Car(String name) {
        this(name, 0);
    }
```
요렇게 주 생성자를 통해 사용할 수 도 있습니다

Reply:
엇 이 방법은 처음 알았어요! 감사합니다

Reply:
헉 이 방법은 처음 알았어요 감사합니다!